source/frontend/src/components/layout/SideNavBar/styles.ts

```diff
@@ -19,7 +19,7 @@ export const SideNavBar = styled.div`
   position: relative;
   align-items: center;
   grid-area: iconbar;
-  background-color: ${props => props.theme.css.primaryColor};
+  background-color: ${props => props.theme.tenant.colour};
```

Comment:
**question:** are we sure this is the correct value? Like this would be the background color for the "Sidebar" and tenant color changes per environment (I believe it is orange in TEST). That would change the current behaviour to the following screenshot the pointed area would turn orange (and purple in UAT). 
That seems a departure of existing functionality. Is this something reviewed by UI/UX?

![image](https://github.com/bcgov/PSP/assets/24322224/a2bb1a04-449d-4cd6-bb9e-aeda71a987a2)